externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/SparkSQLBackendService.scala

```diff
@@ -33,9 +38,40 @@ import org.apache.kyuubi.session.SessionManager
  *               that this backend service holds to run [[org.apache.kyuubi.operation.Operation]]s.
  */
 class SparkSQLBackendService(name: String, spark: SparkSession)
-  extends AbstractBackendService(name) {
+  extends AbstractBackendService(name) with Logging {
 
   def this(spark: SparkSession) = this(classOf[SparkSQLBackendService].getSimpleName, spark)
 
   override val sessionManager: SessionManager = new SparkSQLSessionManager(spark)
+
+  override def renewDelegationToken(
+      sessionHandle: SessionHandle,
+      authFactory: KyuubiAuthenticationFactory,
```

Comment:
where do we use it？

Reply:
Moved method to a new class `SparkThriftFrontendService`.